js/index.js

```diff
@@ -1,7 +1,39 @@
 // Iteration 1: Names and Input
-
+console.log("I'm Ready!")
+let hacker1 = "Johnny";
+console.log(`The driver's name is ${hacker1}`);
+let hacker2 = "Kiyo";
+console.log(`The navigator's name is ${hacker2}`);
 
 // Iteration 2: Conditionals
+if (hacker1.length === hacker2.length) {
+    console.log(`Wow, you both have equally long names, ${hacker1.length} characters!`);
+} else if (hacker1.length > hacker2.length) {
+    console.log(`The driver has the longest name. It has ${hacker1.length} characters.`);
+} else {
+    console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters.`);
+}
+
```

Comment:
Nice job as well here! All good
On a small note, on the conditionals, try to leave the else condition for the "===" in this case, and the other if statements for the "<" and ">" options. As I said the code is still good but it could be more readable for other partners the other way.